### ISSUE TITLE
PFacct: not answering accounting status check ?

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#5279]
````
Apr  1 16:19:10 cluster1 load_balancer[18342]: No response to status check 1808 ID 77 for home server 192.168.126.103 port 1813
Apr  1 16:19:25 cluster1 load_balancer[18342]: No response to status check 1810 ID 5 for home server 192.168.126.102 port 1813
Apr  1 16:19:32 cluster1 load_balancer[18342]: No response to status check 1811 ID 128 for home server 192.168.126.101 port 1813
````

````
2020-04-01 16:20:09.476330 (1) Status-Server Id 141 ens192:192.168.126.100:39679 -> 192.168.126.103:1813 +0.000
	NAS-Identifier = "Status Check 308. Are you alive?"
	Message-Authenticator = 0xc52c0ca8ecb54f1d8df52d23fcaf496a
	Authenticator-Field = 0xc0c744df16bf334a29a30696547db90d
2020-04-01 16:20:09.476456 (1) ** rtx ** Status-Server Id 141 ens224:192.168.126.100:39679 -> 192.168.126.103:1813 +0.000
	NAS-Identifier = "Status Check 308. Are you alive?"
	Message-Authenticator = 0xc52c0ca8ecb54f1d8df52d23fcaf496a
	Authenticator-Field = 0xc0c744df16bf334a29a30696547db90d
2020-04-01 16:20:14.676456 (1) ** norsp ** Status-Server Id 141 ens192:192.168.126.100:39679 -> 192.168.126.103:1813
2020-04-01 16:20:14.676456 (1) Cleaning up request packet ID 141
````

````
radmin> show home_server list
192.168.126.103	1812	udp	auth+acct	unknown	0
192.168.126.103	1813	udp	acct	dead	0
192.168.126.103	1815	udp	auth	unknown	0
192.168.126.102	1812	udp	auth+acct	alive	0
192.168.126.102	1813	udp	acct	dead	0
192.168.126.102	1815	udp	auth	unknown	0
192.168.126.101	1812	udp	auth+acct	alive	0
192.168.126.101	1813	udp	acct	dead	1
192.168.126.101	1815	udp	auth	unknown	0
````

**Comment by jrouzierinverse** [comment]
Fixed by a5a4312e176eaa34f900f8807b6dea0ba92bdfcc

**Comment by lzammit** [comment]
````
radmin> show home_server list
192.168.126.103	1812	udp	auth+acct	unknown	0
192.168.126.103	1813	udp	acct	unknown	0
192.168.126.103	1815	udp	auth	unknown	0
192.168.126.102	1812	udp	auth+acct	alive	0
192.168.126.102	1813	udp	acct	unknown	0
192.168.126.102	1815	udp	auth	unknown	0
192.168.126.101	1812	udp	auth+acct	unknown	0
192.168.126.101	1813	udp	acct	unknown	0
192.168.126.101	1815	udp	auth	unknown	0
````

But I don't see the status checks anymore.

**Comment by jrouzierinverse** [comment]
Fixed

**Comment by lzammit** [comment]
I don't see them.

What should be the frequency of the Status Check between servers ? Thanks.

**Comment by jrouzierinverse** [comment]
I am not sure @fdurand do you know?

**Comment by fdurand** [comment]
15s

**Comment by lzammit** [comment]
It's better I see them going out to the others but they don't answer:

````
[root@cluster1 pf]# radsniff -x -p 1813
Logging all events
Defaulting to capture on all interfaces
Sniffing on (zammitcorp-b ens192 ens224 ens224.145 ens224.146 lo)
2020-04-02 12:56:56.722988 (1) Status-Server Id 33 ens192:192.168.126.100:43456 -> 192.168.126.102:1813 +0.000
	NAS-Identifier = "Status Check 15. Are you alive?"
	Message-Authenticator = 0x3304540d950517207e970bf8162aeaad
	Authenticator-Field = 0x7a4497991fdbebe07243f8206f346b1c
2020-04-02 12:56:56.723061 (1) ** rtx ** Status-Server Id 33 ens224:192.168.126.100:43456 -> 192.168.126.102:1813 +0.000
	NAS-Identifier = "Status Check 15. Are you alive?"
	Message-Authenticator = 0x3304540d950517207e970bf8162aeaad
	Authenticator-Field = 0x7a4497991fdbebe07243f8206f346b1c
2020-04-02 12:56:57.492720 (3) Status-Server Id 178 ens192:192.168.126.100:38652 -> 192.168.126.103:1813 +0.769
	NAS-Identifier = "Status Check 15. Are you alive?"
	Message-Authenticator = 0x1c081a15728577765b5d35907a9dcdba
	Authenticator-Field = 0x3961ea2640aa175079d442f3d785c105
2020-04-02 12:56:57.492821 (3) ** rtx ** Status-Server Id 178 ens224:192.168.126.100:38652 -> 192.168.126.103:1813 +0.769
	NAS-Identifier = "Status Check 15. Are you alive?"
	Message-Authenticator = 0x1c081a15728577765b5d35907a9dcdba
	Authenticator-Field = 0x3961ea2640aa175079d442f3d785c105
2020-04-02 12:56:59.705110 (5) Status-Server Id 56 lo:192.168.126.100:45665 -> 192.168.126.101:1813 +2.982
	NAS-Identifier = "Status Check 15. Are you alive?"
	Message-Authenticator = 0xde5001850875abf32ba7f7f01ea9d758
	Authenticator-Field = 0xd8cac961f5ec8d62e9ceeb091ea01463
2020-04-02 12:57:01.923061 (1) ** norsp ** Status-Server Id 33 ens192:192.168.126.100:43456 -> 192.168.126.102:1813
2020-04-02 12:57:01.923061 (1) Cleaning up request packet ID 33
2020-04-02 12:57:02.692821 (3) ** norsp ** Status-Server Id 178 ens192:192.168.126.100:38652 -> 192.168.126.103:1813
2020-04-02 12:57:02.692821 (3) Cleaning up request packet ID 178
2020-04-02 12:57:04.905110 (5) ** norsp ** Status-Server Id 56 lo:192.168.126.100:45665 -> 192.168.126.101:1813
2020-04-02 12:57:04.905110 (5) Cleaning up request packet ID 56
````

I think it's related to:

````
Apr  2 12:49:16 cluster2 pfacct[4435]: t=2020-04-02T12:49:16-0400 lvl=eror msg="RADIUSSecret: Switch '192.168.126.100' not found :sql: no rows in result set" pid=4435
Apr  2 12:49:22 cluster2 pfacct[4435]: t=2020-04-02T12:49:22-0400 lvl=eror msg="RADIUSSecret: Switch '192.168.126.100' not found :sql: no rows in result set" pid=4435
Apr  2 12:49:52 cluster2 pfacct[4435]: t=2020-04-02T12:49:52-0400 lvl=eror msg="RADIUSSecret: Switch '192.168.126.100' not found :sql: no rows in result set" pid=4435
Apr  2 12:50:24 cluster2 pfacct[4435]: t=2020-04-02T12:50:24-0400 lvl=eror msg="RADIUSSecret: Switch '192.168.126.100' not found :sql: no rows in result set" pid=4435
Apr  2 12:50:56 cluster2 pfacct[4435]: t=2020-04-02T12:50:56-0400 lvl=eror msg="RADIUSSecret: Switch '192.168.126.100' not found :sql: no rows in result set" pid=4435
Apr  2 12:51:26 cluster2 pfacct[4435]: t=2020-04-02T12:51:26-0400 lvl=eror msg="RADIUSSecret: Switch '192.168.126.100' not found :sql: no rows in result set" pid=4435
````

**Comment by jrouzierinverse** [comment]
what is in your cluster.conf

**Comment by lzammit** [comment]
````
[root@cluster1 pf]# cat conf/cluster.conf
[CLUSTER]
management_ip=192.168.126.100

[CLUSTER interface ens192]
ip=192.168.126.100

[CLUSTER interface ens224.145]
ip=172.22.145.4

[CLUSTER interface ens224.146]
ip=172.22.146.4

[cluster1.inverse.ca]
management_ip=192.168.126.101

[cluster1.inverse.ca interface ens192]
ip=192.168.126.101

[cluster1.inverse.ca interface ens224.145]
ip=172.22.145.1

[cluster1.inverse.ca interface ens224.146]
ip=172.22.146.1

[cluster2.inverse.ca]
management_ip=192.168.126.102

[cluster2.inverse.ca interface ens192]
ip=192.168.126.102

[cluster2.inverse.ca interface ens224.145]
ip=172.22.145.2

[cluster2.inverse.ca interface ens224.146]
ip=172.22.146.2

[cluster3.inverse.ca]
management_ip=192.168.126.103

[cluster3.inverse.ca interface ens192]
ip=192.168.126.103

[cluster3.inverse.ca interface ens224.145]
ip=172.22.145.3

[cluster3.inverse.ca interface ens224.146]
ip=172.22.146.3
````

**Comment by jrouzierinverse** [comment]
This should be fixed by e88e956e287400d8bb4474d004694960dc25e706 please retest

**Comment by lzammit** [comment]
Works!
````
[root@cluster1 pf]# radmin -f var/run/radiusd-load_balancer.sock
radmin version 3.0.18, built on Apr 27 2019 at 00:22:18 - FreeRADIUS Server administration tool.
Copyright (C) 2008-2017 The FreeRADIUS server project and contributors.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
You may redistribute copies of FreeRADIUS under the terms of the
GNU General Public License v2.
radmin> show home_server list
192.168.126.103 1812    udp     auth+acct       unknown 0
192.168.126.103 1813    udp     acct    alive   0
192.168.126.103 1815    udp     auth    unknown 0
192.168.126.102 1812    udp     auth+acct       alive   0
192.168.126.102 1813    udp     acct    alive   0
192.168.126.102 1815    udp     auth    unknown 0
192.168.126.101 1812    udp     auth+acct       alive   0
192.168.126.101 1813    udp     acct    alive   0
192.168.126.101 1815    udp     auth    unknown 0
````
